README.md

```diff
@@ -201,3 +201,38 @@ docker logs -f mapper
 Here you should see the message: **Successfully registered mediator!**
 
 Finally, go back to your browser and navigate to the `Mediators` section. Here you should see your mapping mediator instance registered on the OpenHIM with a recent *heartbeat*.
+
+## Kafka Integration
+
+The mediator can consume and produce to [kafka](https://kafka.apache.org/). The kafka broker will have to to set in the [config](./src/config.js) file or via the env variable KAFKA_BLOKERS (default is `localhost:9092`). The topic to consume from (`kafkaConsumerTopic`) or to produce to (`kafkaProducerTopic`) are specified in the endpoint schema as done below.
+
+```js
+{
+  "name": "External Request Test",
+  "endpoint": {
+    "pattern": "/externalRequestTest"
+  },
+  "transformation": {
+    "input": "JSON",
+    "output": "JSON"
+  },
+  "kafkaConsumerTopic": "2xx",
+  "requests": {
+    "response": [
+      {
+        "id": "fhirPatient",
+         "kafkaProducerTopic": "3xx"
+      }
+    ]
+  }
+}
+```
+
+There are scripts for setting up a kafka instance locally. Run following script in the root folder
+
+```sh
+docker-compose -f docker-compose.kafka.yml up -d
+
+# To create topics got to the root folder of the running container and run
```

Comment:
```suggestion
# To create topics go to the root folder of the running container and run
```

Reply:
addressed in https://github.com/jembi/openhim-mediator-mapping/pull/205/commits/dd30cd98f267f8a46b99a1cb686be24a5a90613c